colon-kw/lang/reader.rkt

```diff
@@ -32,5 +33,9 @@
      (lambda (key defval)
        (define (fallback) (if proc (proc key defval) defval))
        (case key
+         [(color-lexer)
+          (make-colon-kw-lexer
+           (or (fallback)
```

Comment:
I think you’re missing some of what’s going on in `lexer.rkt`—it explicitly imports `syntax-color/racket-lexer` there, too, and it calls out to it directly rather than using `orig-lexer` on lines 12 and 17. It *really is* independent of the base lexer there.